nodes/cma/nodes-cma-rn/nodes-cma-autoscaling-custom-rn-past.adoc

```diff
@@ -10,6 +10,23 @@ The following release notes are for previous versions of the Custom Metrics Auto
 
 For the current version, see xref:../../../nodes/cma/nodes-cma-rn/nodes-cma-autoscaling-custom-rn.adoc#nodes-cma-autoscaling-custom-rn[Custom Metrics Autoscaler Operator release notes].
 
+[id="nodes-pods-autoscaling-custom-rn-2121-376_{context}"]
+== Custom Metrics Autoscaler Operator 2.12.1-376 release notes
+
+This release of the Custom Metrics Autoscaler Operator 2.12.1-376 provides security updates and bug fixes for running the Operator in an {product-title} cluster. The following advisory is available for the link:https://access.redhat.com/errata/RHSA-2024:1812[RHSA-2024:1812].
+
+[IMPORTANT]
+====
+Before installing this version of the Custom Metrics Autoscaler Operator, remove any previously installed Technology Preview versions or the community-supported version of KEDA.
+====
+
+[id="nodes-pods-autoscaling-custom-rn-2121-376-bugs_{context}"]
+=== Bug fixes
+
+* Previously, if invalid values, such as nonexistent namespaces, were specified in scaled object metadata, the underlying scaler clients would not free, or close, their client descriptors, resulting in a slow memory leak. This fix properly closes the underlying client descriptors when there are errors, preventing memory from leaking. (link:https://issues.redhat.com/browse/OCPBUGS-30145[*OCPBUGS-30145*])
```

Comment:
```suggestion
* Previously, if invalid values such as nonexistent namespaces were specified in scaled object metadata, the underlying scaler clients would not free or close their client descriptors, resulting in a slow memory leak. This fix properly closes the underlying client descriptors when there are errors, preventing memory from leaking. (link:https://issues.redhat.com/browse/OCPBUGS-30145[*OCPBUGS-30145*])
```

Reply:
I _think_ this reads smoother; however, I understand that a few of these commas are for non-restrictive clauses. I leave it to you.